### PR TITLE
Create View for User Login/Register

### LINKED ISSUE #67: Create view for "User Login Screen"
(UI Design Reference)[https://github.com/CMPUT301F21T09/BudgetProjectName/wiki/User-Interface-Mockups#login-screen]

See #44

### PR DESCRIPTION
**Pull request by Docker-J**
## Description of Changes
Created View for Login/Register
Fix little errors that occurred from previous merges 

TODO: I made custom xml Background for UserLoginActivity to fit various size of screen but UI looks bad as background streches

## Screenshots
LoginRegister
Screenshot 2021-11-02 123120

## Checklist

- [ ] Automated tests
- [x] Screenshots included (if necessary)
- [x] Code is well commented

Closes #67, #68

### CODE REVIEW

**Review by AtrazineC — APPROVED** [review]
Great job!

**Review by loravocado — APPROVED** [review]
That looks amazing! I love it! Good work ❤️

File: src/app/src/main/res/drawable/login_background.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            android:toDegrees="-30">
+            <shape android:shape="rectangle">
+                <solid android:color="@color/white" />
+            </shape>
+        </rotate>
+    </item>
+
+</layer-list>
+
+<!--<layer-list xmlns:android="http://schemas.android.com/apk/res/android">-->
```
Comment (MacMalainey) [inline src/app/src/main/res/drawable/login_background.xml:35]: Is there any reason this commented out code still needs to be here?

**Review by MacMalainey — COMMENTED** [review]
One nit, otherwise LGTM

**Pull request merged**